source/usage-examples/find.txt

```diff
@@ -25,8 +25,8 @@ Click `here <{+example+}/find.go>`__ to see a fully runnable example.
 Expected Result
 ---------------
 
-This example creates a ``Cursor`` object that returns
-documents similar to these:
+After you run the preceding example, it returns a ``Cursor`` object that
+contains the following documents:
```

Comment:
I think the original text was correct in  saying that the Cursor returns documents. It does not contain documents.

Reply:
Will change "that contains" to "with"

Reply:
I'll change this to say "...  it creates a ``Cursor`` object that returns the following documents:"